templates/automotive/index.html

```diff
@@ -220,43 +65,348 @@ <h2 class="p-muted-heading u-align--center u-sv3">THESE AUTOMOTIVE INNOVATORS HA
+      <p>Full OS containerisation means transactional, atomic updates. Data snapshot backup and restore. Signed software integrity. <a href="/engage/snap-deltas">Compressed delta updates</a>.</p>
+      <p><a href="/engage/snapd-whitepaper">Complete control</a>. No delayed patches, no partial updates, no unauthorised ECU software.</p>
+    </div>
+    <div class="col-6 u-vertically-center u-align--center u-hide--small">
+      {{
+        image(
+          url="https://assets.ubuntu.com/v1/74859dc8-control-thin.svg",
+          alt="",
+          height="200",
+          width="205",
+          hi_def=True,
+        ) | safe
+      }}
+    </div>
+  </div>
+</section>
+
+<section class="p-strip">
+  <div class="row u-equal-height">
+    <div class="col-6 u-vertically-center u-align-center u-hide--small">
+      {{
+        image(
+          url="https://assets.ubuntu.com/v1/0be66895-Automotive-car-sensor.svg",
+          alt="",
+          height="200",
+          width="350",
+          hi_def=True,
+        ) | safe
+      }}
+    </div>
+    <div class="col-6">
+      <h2 class="p-heading--five">V2X communication</h2>
+      <p class="p-heading--two">Cruise control for private cloud</p>
+      <p>Orchestrate <a href="/telco">V2X and NFV on carrier-grade</a> clouds.</p>
+      <p>Own the edge with <a href="https://maas.io" class="p-link--external">Metal as a Service</a>.</p>
+      <p>Connect and containerise with <a href="/kubernetes">Kubernetes</a> from vehicle to edge.</p>
+      <p>Crisp, economical, robust and scalable.</p>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip--square-suru u-image-position" style="overflow: visible;">
+  <div class="row u-equal-height">
+    <div class="col-6">
+      <h2 class="p-heading--five">Mixed-criticality ECUs</h2>
+      <p class="p-heading--two">RTOS safety, Ubuntu agility</p>
+      <p>Isolate safety-critical software, and run consumer-friendly workloads on Ubuntu.</p>
+      <p>Ubuntu delivers <a href="/engage/451-automotive" class="p-link--inverted">software-defined differentiation</a>. Run AUTOSAR Adaptive. Gather telemetry. Manage batteries. Connect customers.</p>
+      <p>Now available as <a href="https://projectacrn.github.io/latest/getting-started/rt_industry_ubuntu.html" class="p-link--inverted p-link--external">open source ACRN</a>.</p>
+      <p>Crisp, economical, robust and scalable.</p>
+    </div>
+    <div class="col-6 u-vertically-center u-align-center u-hide--small">
+      {{
+        image(
+          url="https://assets.ubuntu.com/v1/9c4d4439-Automotive-car-chassi-new.svg",
+          alt="",
+          height="284",
+          width="422",
+          hi_def=True,
+          attrs={"class": "u-image-position--bottom", "style": "bottom: -3rem"},
+        ) | safe
+      }}
+    </div>
+  </div>
+</section>
+
+<section class="p-strip">
+  <div class="row">
+    <div class="col-6">
+      <h2 class="p-heading--five">ARM and X86</h2>
```

Comment:
you are missing the arm/nvidia/intel logos

Reply:
Yeah, waiting on design to provide the images.